src/main/java/app/AppBuilder.java

```diff
@@ -1,52 +1,133 @@
 package app;
 
-import view.gui_components.TradeSimulationFrame;
+import data_access.InMemoryStockDataAccessObject;
```

Comment:
To simplify things a little bit, we may use  `import data_access.* ` etc. for the `import` statements, since we will need to import classes for every use case and every view component anyways.

Reply:
I'm using IntelliJ to optimize imports gloablly, later on it may be automatically changed to what you said.

Reply:
Also I'm thinking about how to automatically grab all these stuff during building phase, I'll add a todo here.